sources/editor/Xenko.Assets.Presentation/Templates/Core/ProjectExecutable.Windows/$ProjectName$.csproj.t4

```diff
@@ -2,11 +2,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net461</TargetFramework>
+    <!-- We should switch Windows Game to netCore as well. Would help us find xplatform issues quicker -->
```

Comment:
Let's keep Windows as is.
It's a big breaking change to convert all users Windows project from .NET Framework to .NET Core.
Note that I am not against the concept. However, we'll likely have to do this in a separate PR with some backward compat options (i.e. offer a .NET Core/Framework dropdown when creating the Windows platform).

Reply:
Is there a way to have both configuration target generated, and keep Windows as default?

Something like `<TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>` should work but I think it builds both targets by default (?), which we might want to avoid.

Reply:
I think it's fine to have two separate projects (either simultaneously with an additional Xenko platform or just a dropdown choice for Windows platform).

Otherwise it would lead to lot of complexity: since the project have some different prorperties, there would be lot of Condition depending on the `TargetFramework` (that will get destroyed once people edit their project in VS).
And I don't see much advantages over having explicit simple projects that the user can generate/regenerate at will.

So my opinion: let's keep it simple (no MSBuild target-hell like on Xenko dev side), and similar to what VS generates out of the box.

Reply:
Since this is just the template for new projects this should not break existing projects right? 

My main concern here is that by not having netcore as the default moving forward we do not see/fix issues on that platform (which is key for Mac/Linux) until later, and increases maintenance burden for us. A question perhaps for the Developer meetup is how to move forward? My vote is to go 100% NetCore on platforms that can support that. There's many advantages to that. And with NetCore3 on the horizon there is improvements for GUI/Window apps.

Regarding short term solution: What about a new Platform in the Project creation that is - NetCore, this could then support all 3 desktop platforms via Multi-targeting. Then we leave existing Windows (.NetFramework) alone. How does that sound? Then user can decide, which or both if they want. I think this what you guys are alluding to above.

Reply:
Sure, I am also all for switching to .NET Core as soon as possible.
However, this probably warrants a discussion on how/when to do it so I would prefer to keep that out of this PR to be able to merge that one quickly.